src/itest/resources/features/psc_statements_response_codes.feature

```diff
@@ -10,7 +10,6 @@ Feature: Response codes for psc statements
 
     Examples:
     | data                                   | response_code |
-    | bad_company_psc_statement_no_delta_at  | 400           |
```

Comment:
Could we have a new test for confirm that no delta_at gets persisted in mongo?

Reply:
I added unit tests for it instead, not sure we need an itest for it as well